ntoskrnl/io/iomgr/deviface.c

```diff
@@ -304,7 +305,191 @@ IoGetDeviceInterfaceAlias(IN PUNICODE_STRING SymbolicLinkName,
                           IN CONST GUID *AliasInterfaceClassGuid,
                           OUT PUNICODE_STRING AliasSymbolicLinkName)
 {
-    return STATUS_NOT_IMPLEMENTED;
+    UNICODE_STRING SymbolicLink, AliasSymbolicLink = {0, 0, NULL};
+    UNICODE_STRING DosDevicesPrefix1 = RTL_CONSTANT_STRING(L"\\??\\");
+    UNICODE_STRING DosDevicesPrefix2 = RTL_CONSTANT_STRING(L"\\\\?\\");
```

Comment:
Is that one possible? In pure NT kernel mode, shouldn't that one already been filtered (since it's Win32 and not NT) and converted to NT path (a `\\??\\` prefix) ?